src/inference/AdvancedSMC.jl

```diff
@@ -118,11 +106,8 @@ function sample_init!(
     # create a new particle container
     spl.state.particles = pc = ParticleContainer(model, particles)
 
-    while consume(pc) !== Val{:done}
-        ess = effectiveSampleSize(pc)
-        if ess <= spl.alg.resampler_threshold * N
-            resample!(pc, spl.alg.resampler)
-        end
+    while consume(spl.state.particles) !== Val{:done}
```

Comment:
Either use `pc` here or remove the `pc` definition above.

Reply:
Oh, I guess I missed that when I merged the master branch.